embedded/remotestorage/s3/s3.go

```diff
@@ -73,12 +80,20 @@ var (
 	ErrInvalidResponseSubPathUnescape      = fmt.Errorf("%w: error un-escaping object name", ErrInvalidResponse)
 
 	ErrTooManyRedirects = errors.New("too many redirects")
+
+	arnRoleRegex = regexp.MustCompile(`arn:.*\/(.*)`)
```

Comment:
would this regex match all possible role definitions?

Reply:
This one is quite wide, so I believe that this regex will catch any role.